staging/src/k8s.io/apiserver/pkg/apis/flowcontrol/bootstrap/default.go

```diff
@@ -0,0 +1,336 @@
+		1000, // matchingPrecedence
+		"",   // distinguisherMethodType
+		flowcontrol.PolicyRulesWithSubjects{
+			Subjects: groups(user.SystemPrivilegedGroup),
+			ResourceRules: []flowcontrol.ResourcePolicyRule{
+				resourceRule(
+					[]string{flowcontrol.VerbAll},
+					[]string{flowcontrol.APIGroupAll},
+					[]string{flowcontrol.ResourceAll},
+					[]string{flowcontrol.NamespaceEvery},
+					true,
+				),
+			},
+			NonResourceRules: []flowcontrol.NonResourcePolicyRule{
+				nonResourceRule(
+					[]string{flowcontrol.VerbAll},
+					[]string{flowcontrol.NonResourceAll},
+				),
+			},
+		},
+	)
+	DefaultFlowSchemaDefault = newFlowSchema(
+		"default",
+		"default",
+		10000, // matchingPrecedence
+		flowcontrol.FlowDistinguisherMethodByUserType, // distinguisherMethodType
+		flowcontrol.PolicyRulesWithSubjects{
+			Subjects: groups(user.AllUnauthenticated, user.AllAuthenticated),
+			ResourceRules: []flowcontrol.ResourcePolicyRule{
+				resourceRule(
+					[]string{flowcontrol.VerbAll},
+					[]string{flowcontrol.APIGroupAll},
+					[]string{flowcontrol.ResourceAll},
+					[]string{flowcontrol.NamespaceEvery},
+					true,
+				),
+			},
+			NonResourceRules: []flowcontrol.NonResourcePolicyRule{
+				nonResourceRule(
+					[]string{flowcontrol.VerbAll},
+					[]string{flowcontrol.NonResourceAll},
+				),
+			},
+		},
+	)
+)
+
+// PredefinedPriorityLevelConfigurations returns the instances of
+// the ones defined in the KEP
+func PredefinedPriorityLevelConfigurations() []*flowcontrol.PriorityLevelConfiguration {
+	return []*flowcontrol.PriorityLevelConfiguration{
+		// system priority-level
+		newPriorityLevelConfiguration("system",
+			flowcontrol.PriorityLevelConfigurationSpec{
+				Type: flowcontrol.PriorityLevelEnablementLimited,
+				Limited: &flowcontrol.LimitedPriorityLevelConfiguration{
+					AssuredConcurrencyShares: 30,
+					LimitResponse: flowcontrol.LimitResponse{
+						Type: flowcontrol.LimitResponseTypeQueue,
+						Queuing: &flowcontrol.QueuingConfiguration{
+							Queues:           64,
+							HandSize:         6,
+							QueueLengthLimit: 1000,
+						},
+					},
+				},
+			}),
+		// workload-high priority-level
+		newPriorityLevelConfiguration("workload-high",
+			flowcontrol.PriorityLevelConfigurationSpec{
+				Type: flowcontrol.PriorityLevelEnablementLimited,
+				Limited: &flowcontrol.LimitedPriorityLevelConfiguration{
+					AssuredConcurrencyShares: 30,
+					LimitResponse: flowcontrol.LimitResponse{
+						Type: flowcontrol.LimitResponseTypeQueue,
+						Queuing: &flowcontrol.QueuingConfiguration{
+							Queues:           128,
+							HandSize:         6,
+							QueueLengthLimit: 100,
+						},
+					},
+				},
+			}),
+		// workload-low priority-level
+		newPriorityLevelConfiguration("workload-low",
+			flowcontrol.PriorityLevelConfigurationSpec{
+				Type: flowcontrol.PriorityLevelEnablementLimited,
+				Limited: &flowcontrol.LimitedPriorityLevelConfiguration{
+					AssuredConcurrencyShares: 30,
```

Comment:
I'd expect `workload-low` to have at most half as many shares as `workload-high`?

Reply:
updated. now the workload-high has 40 and the workload-low has 20, that makes the latter half as the former. wdyt?